examples/ceed/ex2-surface.c

```diff
@@ -425,7 +425,6 @@ int SetCartesianMeshCoords(int dim, int nxyz[3], int mesh_order,
 
 #ifndef M_PI
 #define M_PI    3.14159265358979323846
-#define M_PI_2  1.57079632679489661923
```

Comment:
Thanks for your revision Jeremy. What do you mean exactly by the change using it?  I have just removed the declaration of the `M_PI_2` define because it was unused in the code.

Reply:
I worded it poorly. I just meant that it looked unused.